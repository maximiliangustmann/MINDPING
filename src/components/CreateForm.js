import React from 'react'
import styled from 'styled-components/macro'

export default function CreateForm() {
  return (
    <Form>
      <label>
        Title
        <Input></Input>
      </label>
      <label>
        Rating
        <Input></Input>
      </label>
      <label>
        Categorie
        <Input></Input>
      </label>
      <label>
        Subcategorie
        <Input></Input>
      </label>
      <label>
        Summary
        <Input></Input>
      </label>
      <label>
        Lesson
        <Input></Input>
      </label>
      <ButtonGroup>
        <button>Cancel</button>
        <button>Submit</button>
      </ButtonGroup>
    </Form>
  )
}

const Form = styled.form`
  display: grid;
  gap: 20px;
  align-content: start;
  margin: 20px;
  padding: 20px;
  width: 100px;
`
const Input = styled.input`
  margin: 0 20px;
`

const ButtonGroup = styled.div`
  display: flex;
  justify-content: space-evenly;
`
