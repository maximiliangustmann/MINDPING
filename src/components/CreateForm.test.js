import React from 'react'
import renderer from 'react-test-renderer'
import CreateForm from './CreateForm'
import { MemoryRouter } from 'react-router-dom'

describe('CreateForm', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <MemoryRouter>
        <CreateForm />
      </MemoryRouter>
    )
    expect(tree).toMatchSnapshot()
  })
})
