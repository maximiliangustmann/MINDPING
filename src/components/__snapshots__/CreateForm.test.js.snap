// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateForm renders correctly 1`] = `
<form
  className="CreateForm__Form-sc-1sav95a-0 iwltNh"
  onSubmit={[Function]}
>
  <label>
    Title
    <input
      autoFocus={true}
      className="CreateForm__Input-sc-1sav95a-1 WCGvC"
      name="title"
    />
  </label>
  <label>
    Rating
    <input
      className="CreateForm__Input-sc-1sav95a-1 WCGvC"
      name="rating"
    />
  </label>
  <label>
    Categorie
    <input
      className="CreateForm__Input-sc-1sav95a-1 WCGvC"
      name="categorie"
    />
  </label>
  <label>
    Subcategorie
    <input
      className="CreateForm__Input-sc-1sav95a-1 WCGvC"
      name="subcategorie"
    />
  </label>
  <label>
    Summary
    <input
      className="CreateForm__Input-sc-1sav95a-1 WCGvC"
      name="summary"
    />
  </label>
  <label>
    Lesson
    <input
      className="CreateForm__Input-sc-1sav95a-1 WCGvC"
      name="lessons"
    />
  </label>
  <div
    className="CreateForm__ButtonGroup-sc-1sav95a-2 grXHUY"
  >
    <a
      aria-current="page"
      className="active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      <button>
        Cancel
      </button>
    </a>
    <button
      type="submit"
    >
      Submit
    </button>
  </div>
</form>
`;
